ui/components/src/Table/TableRow.tsx

```diff
@@ -25,9 +25,9 @@ export const TableRow = forwardRef<HTMLTableRowElement, TableRowProps>(function
       {...props}
       ref={ref}
       sx={{
-        backgroundColor: (theme) => theme.palette.background.paper,
+        backgroundColor: (theme) => theme.palette.background.default,
         '&:hover': {
-          backgroundColor: (theme) => theme.palette.primary.light,
+          backgroundColor: (theme) => alpha(theme.palette.primary.main, theme.palette.action.hoverOpacity),
```

Comment:
While testing the general bg color changes, I realized the color I selected here for table row hover didn't align well with dark mode or the list variation of the legend, so I adjusted it a little bit to be more in sync.